src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/ExportTab.php

```diff
@@ -146,74 +160,55 @@ private function renderExport()
         $this->renderFormEnd();
     }
 
-    private function handleExport(string $type, string $lang)
+    private function renderHelp(): void
     {
-        $typeName = FileExportTypeHelper::getTypePluralName($type);
-        $title = sprintf(
-            __('Please wait and keep the page open while we are exporting your %s.', I18N::DOMAIN),
-            strtolower($typeName)
-        );
-        $description = __('your export will be downloaded as soon the export finished.', I18N::DOMAIN);
-
-        $overlay = new OverlayRenderer();
-        $overlay->start($title, $description);
-
-        try {
-            $url = $this->runExport($type, $lang, $overlay);
-
-            $this->initializeDownload($url);
-
-            sleep(1); // wait for the download to start client sided
+        $documentationText = __('See documentation', I18N::DOMAIN);
+
+        $guides = [
+            __('Check if `wp-cli` is installed in the website\'s server.', I18N::DOMAIN).
+            " <a target='_blank' href='".BackofficeCore::DOCS_WPCLI_LINK."'>{$documentationText}</a>",
+            __('Open command line and navigate to website directory', I18N::DOMAIN).': <code>cd '.ABSPATH.'</code>',
+            sprintf(__('Run %s to export %s.', I18N::DOMAIN), '<code>wp sk file-export-all</code>', __('full package')),
+            [
+                'parent' => __('or alternatively, you can export per file.', I18N::DOMAIN),
+                'children' => [
+                    sprintf(__('Run %s to export %s.', I18N::DOMAIN), '<code>wp sk file-export-customer</code>', __('customers')),
+                    sprintf(__('Run %s to export %s.', I18N::DOMAIN), '<code>wp sk file-export-tag</code>', __('tags')),
+                    sprintf(__('Run %s to export %s.', I18N::DOMAIN), '<code>wp sk file-export-category</code>', __('categories')),
+                    sprintf(__('Run %s to export %s.', I18N::DOMAIN), '<code>wp sk file-export-attribute</code>', __('attributes')),
+                    sprintf(__('Run %s to export %s.', I18N::DOMAIN), '<code>wp sk file-export-attribute-option</code>', __('attribute options')),
+                    sprintf(__('Run %s to export %s.', I18N::DOMAIN), '<code>wp sk file-export-product-blueprint</code>', __('product blueprints')),
+                    sprintf(__('Run %s to export %s.', I18N::DOMAIN), '<code>wp sk file-export-product</code>', __('products')),
+                ],
+            ],
+            __('Each run will return the path of the exported file.', I18N::DOMAIN),
+        ];
```

Comment:
those translations should be escaped, cos you directly show the in html

Reply:
Used `esc_html__` for translations.